modules/addons/transports/http/undertow/src/main/java/org/jboss/wsf/stack/cxf/addons/transports/undertow/UndertowServerEngineFactory.java

```diff
@@ -97,9 +95,7 @@ public synchronized UndertowServerEngine retrieveHttpServerEngine(int port)
    public synchronized UndertowServerEngine createHttpServerEngine(String host, int port, String protocol)
          throws IOException
    {
-      if (LOG.isLoggable(Level.FINE)) {
-         LOG.fine("Creating HttpServer Engine for port " + port + ".");
-      }
+      LOG.debug("Creating HttpServer Engine for port " + port + ".");
```

Comment:
Not really a major issue, but considering you're doing a string concat to create the message here, you need to keep the if clause to check if the logs are going to be printed or not (isDebugEnabled)